src/extra/Markdown/index.tsx

```diff
@@ -1,575 +1,60 @@
-		color: #032f62;
-	}
-
-	.octicon {
-		display: inline-block;
-		vertical-align: text-top;
-		fill: currentColor;
-	}
-
-	a {
-		background-color: transparent;
-		-webkit-text-decoration-skip: objects;
-	}
-
-	a:active,
-	a:hover {
-		outline-width: 0;
-	}
-
-	strong {
-		font-weight: inherit;
-	}
-
-	strong {
-		font-weight: bolder;
-	}
-
-	h1 {
-		font-size: 2em;
-		margin: 0.67em 0;
-	}
-
-	img {
-		border-style: none;
-	}
-
-	svg:not(:root) {
-		overflow: hidden;
-	}
-
-	code,
-	kbd,
-	pre {
-		font-family: ${(props) => props.theme.monospace};
-		font-size: 1em;
-	}
-
-	hr {
-		box-sizing: content-box;
-		height: 0;
-		overflow: visible;
-	}
-
-	input {
-		font: inherit;
-		margin: 0;
-	}
-
-	input {
-		overflow: visible;
-	}
-
-	[type='checkbox'] {
-		box-sizing: border-box;
-		padding: 0;
-	}
-
+import Heading from '../../components/Heading';
+import Link from '../../components/Link';
+import unified from 'unified';
+import remarkParse from 'remark-parse';
+import remarkRehype from 'remark-rehype';
+import rehypeReact from 'rehype-react';
+import sanitize from 'rehype-sanitize';
+import raw from 'rehype-raw';
+import prism from '@mapbox/rehype-prism';
+import Divider from '../../components/Divider';
+import styled, { withTheme } from 'styled-components';
+import gh from 'hast-util-sanitize/lib/github.json';
+import { Theme } from '../../common-types';
+import 'prismjs/themes/prism.css';
+export { gh as defaultSanitizerOptions };
+
+const MarkdownWrapper = styled(Txt)`
 	* {
 		box-sizing: border-box;
 	}
 
-	input {
-		font-family: inherit;
-		font-size: inherit;
-		line-height: inherit;
-	}
-
-	a {
-		color: #0366d6;
-		text-decoration: none;
-	}
-
-	a:hover {
-		text-decoration: underline;
-	}
-
-	strong {
-		font-weight: 600;
-	}
-
-	hr {
-		height: 0;
-		margin: 15px 0;
-		overflow: hidden;
-		background: transparent;
-		border: 0;
-		border-bottom: 1px solid #dfe2e5;
-	}
-
-	hr::before {
-		display: table;
-		content: '';
-	}
-
-	hr::after {
-		display: table;
-		clear: both;
-		content: '';
-	}
-
-	table {
-		border-spacing: 0;
-		border-collapse: collapse;
-	}
-
-	td,
-	th {
-		padding: 0;
-	}
-
-	h1,
-	h2,
-	h3,
-	h4,
-	h5,
-	h6 {
-		margin-top: 0;
-		margin-bottom: 0;
-	}
-
-	h1 {
-		font-size: 32px;
-		font-weight: 400;
-	}
-
-	h2 {
-		font-size: 24px;
-		font-weight: 400;
-	}
-
-	h3 {
-		font-size: 20px;
-		font-weight: 400;
-	}
-
-	h4 {
-		font-size: 16px;
-		font-weight: 400;
-	}
-
-	h5 {
-		font-size: 14px;
-		font-weight: 400;
-	}
-
-	h6 {
-		font-size: 12px;
-		font-weight: 400;
-	}
-
-	p {
-		margin-top: 0;
-		margin-bottom: 10px;
-	}
-
-	blockquote {
-		margin: 0;
-	}
-
-	ul,
-	ol {
-		padding-left: 0;
-		margin-top: 0;
-		margin-bottom: 0;
-	}
-
-	ol ol,
-	ul ol {
-		list-style-type: lower-roman;
-	}
-
-	ul ul ol,
-	ul ol ol,
-	ol ul ol,
-	ol ol ol {
-		list-style-type: lower-alpha;
-	}
-
-	dd {
-		margin-left: 0;
-	}
-
-	code {
-		font-size: 1rem;
+	& code {
+		background-color: rgba(27, 31, 35, 0.05);
```

Comment:
Any particular reason for this color? I notice from the screenshot files that it's a bit more 'yellowish' than the previous codeblock background.

Reply:
This is exactly the same color used before. the yellowish tint might be due to prismjs